api/src/wfl/util.clj

```diff
@@ -562,6 +562,16 @@
   (let [[name value & rest] (str/split s #":" 3)]
     (and (label-name? name) (label-value? value) (nil? rest))))
 
+(defn label-value
+  "Return first value associated with `label-name` in `labels`
+  or nil if no match."
+  [labels label-name]
+  (when-let [matched (->> labels
+                          (filter #(str/starts-with? % (str label-name ":")))
+                          first)]
+    (let [[_name value & _rest] (str/split matched #":" 3)]
+      value)))
```

Comment:
Might also deconstruct labels into a map but OK.
Or maybe repeated keys are allowed or something.

Reply:
Yeah, I think repeated keys are allowed (for staged workloads at least):

https://github.com/broadinstitute/wfl/blob/develop/api/src/wfl/module/covid.clj#L68-L73